src/ui/server/test-mocks/mock-eligibility.ts

```diff
@@ -13,6 +13,29 @@ const mockEligibility = {
   totalValueInsured: 3,
   coverPeriod: COVER_PERIOD.LESS_THAN_2_YEARS.DB_ID,
   totalContractValue: TOTAL_CONTRACT_VALUE.LESS_THAN_500K.DB_ID,
+  company: {
+    id: 'clcyyopn40148m8noyar9wxrn',
+    companyName: 'Test Name',
```

Comment:
Can we have "Mock Company" or something that mentions company.

Just thinking of debugging the XLSX, will be easier to check things